NPM Install Failing

Hi Jeremy/Joints team,

I've been using Joints for several years now without issue. Suddenly on npm install, Terminal throws back:

`npm ERR! code 1
npm ERR! Command failed: /usr/bin/git checkout 4.0
npm ERR! error: pathspec '4.0' did not match any file(s) known to git.`

Any idea what that's all about?

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "babel-preset-es2015": "^6.5.0",
     "browser-sync": "^2.11.0",
     "foundation-sites": "6.4.3",
-    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp": "github:gulpjs/gulp#v4.0.0",
     "gulp-autoprefixer": "^2.3.1",
     "gulp-babel": "^6.1.2",
     "gulp-concat": "^2.5.2",

